README.md

```diff
@@ -1,3 +1,273 @@
+  * [Viewing kubeconfig (GKE)](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl#viewing_kubeconfig)
+  * [Create a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html)
+  * [Use Azure role-based access controls to define access to the Kubernetes configuration file in Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/control-kubeconfig-access)
+
+## Procedure
+
+1. Clone the `sow` repository and build a Docker image that contains the command line tool [sow](https://github.com/gardener/sow) and all tools `sow` requires. Once the image is built and you have added the path to our [wrapper script](https://github.com/gardener/sow/blame/master/docker/bin/sow) to your `PATH` variable you can call `sow` on the command line.
+
+    ```bash
+    # setup for calling sow via the wrapper
+    git clone "https://github.com/gardener/sow"
+    cd sow
+    make # will build and tag the image
+    export PATH=$PATH:$PWD/docker/bin
+    ```
+
+    > The wrapper script starts `sow` in a Docker container, executes the sow command with the given arguments, and then exits and removes the container again. The script also mounts parts of your file system into that container to allow `sow` to persist the state in directory `landscape` which you will create in the next step.
+
+1. Create a directory for your Gardener landscape and clone this repository into a subdirectory called `crop`:
+
+    ```bash
+    mkdir landscape
+    cd landscape
+    git clone "https://github.com/gardener/garden-setup" crop
+    ```
+
+1. In your landscape folder, create a configuration file called `acre.yaml`. The structure of the configuration file is described [below](#configuration-file-acreyaml).
+
+    > Do not use file `acre.yaml` in folder `crop`. This file is used internally by the installation tool.
+
+1. If you created the base cluster using GKE you need to convert your `KUBECONFIG` to one that uses basic authentication with Google-specific configuration parameters:
+
+    1. Set your `KUBECONFIG` environment variable to a file that does not exist. Do *not* use the one specified in your acre.yaml file.
+
+    1. Download the GKE kubeconfig:
+
+        ```bash
+        gcloud container clusters get-credentials <your_cluster> --zone <your_zone> --project <your_project>
+        ```
+
+    1. Call `crop/bin/convertkubeconfig 3.1`. When asked for credentials, enter the ones that the GKE dashboard shows when clicking on `show credentials`.
+
+    This will create a kubeconfig that uses basic auth. It will be created at the path specified in your `acre.yaml` file.
+
+1. Open a second terminal window and enter the following command to watch the progress of the Gardener installation:
+
+    ```bash
+    watch -d kubectl -n garden get pods,ingress,sts,svc
+    ```
+
+1. In your first terminal window, use the following command to deploy Gardener on your base cluster:
```

Comment:
It's probably a good idea to mention a `sow order -A` before. The command will detect some kinds of errors in the acre.yaml (if the user made some), but it doesn't deploy something. Also, the user than has an idea about the amount and order of components.

Reply:
done